src/main/python/msmarco/convert_trec_to_msmarco_run.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+Anserini: A Lucene toolkit for replicable information retrieval research
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import argparse
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Converts a TREC run file to a MS MARCO-formatted run file.')
+    parser.add_argument('--input_run', required=True, default='', help='TREC-formatted run file')
+    parser.add_argument('--output_run', required=True, default='',
+                        help='output MS MARCO-formatted run file')
+
+    args = parser.parse_args()
+
+    with open(args.output_run, 'w') as fout:
+        for line in open(args.input_run):
+            query_id, _, doc_id, rank, _, _ = line.strip().split(' ')
+            fout.write('{}\t{}\t{}\n'.format(
+                query_id, doc_id, rank))
+
+    print('Done!')
```

Comment:
Maybe something more informative, like "Wrote output run to ...".

Reply:
Also is there any reason to assume that the run file is sorted by rank? Because I feel like in most of our eval scripts we sort it again by rank regardless.

Reply:
Yes, a check to make sure the ranks are strictly increase and scores are strictly decreasing would be good, and throw a warning.

Score ties are a hidden gotcha, and this will expose.

Reply:
Sure